docs/contributing/decisions/0010-account-record-table/prototypes/01-width-diagnosis.html

```
@@ -0,0 +1,1704 @@
+<title>Account detail density — four ways out</title>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)